app/models/info_request.rb

```diff
@@ -55,9 +55,12 @@ class InfoRequest < ApplicationRecord
                    :replace_newlines => true, :collapse_spaces => true
 
   validates_presence_of :title, :message => N_("Please enter a summary of your request")
-  validates_format_of :title, :with => /[[:alpha:]]/,
-    :message => N_("Please write a summary with some text in it"),
-    :unless => Proc.new { |info_request| info_request.title.blank? }
+
+  validates_format_of :title,
+    with: /\A.*[[:alpha:]]+.*\z/,
```

Comment:
I don't agree in this case. I think there are a few options, but the current implementation is best.

1. All parameters indented. This is pretty gross as to find the validation you're looking for, your eyes would have to jump up and down the first and second lines. Grepping would also be difficult, as you'd have to do something like `grep -A 1 validates_`, and then probably remove the newline to get a nice inline listing of validations.

```ruby
  validates_format_of
    :title,
    with: /\A.*[[:alpha:]]+.*\z/,
    message: N_('Please write a summary with some text in it'),
    unless: proc { |info_request| info_request.title.blank? }
```

2. All parameters aligned. This causes a particular problem with validations because the line length is quickly exceeded by the `:message` option.

```ruby
  validates_format_of :title,
                      with: /\A.*[[:alpha:]]+.*\z/,
                      message: N_('Please write a summary with some text in it'),
                      unless: proc { |info_request| info_request.title.blank? }
```

You then end up with this type of thing:

```ruby
  validates_format_of :title,
                      with: /\A.*[[:alpha:]]+.*\z/,
                      message:
                        N_('Please write a summary with some text in it'),
                      unless: proc { |info_request| info_request.title.blank? }
```

So, I think I'd propose keeping the implementation as is. I see these more akin to multi-line method chaining:

```ruby
  validates_format_of :title,
    with: /\A.*[[:alpha:]]+.*\z/,
    message: N_('Please write a summary with some text in it'),
    unless: proc { |info_request| info_request.title.blank? }

  validates_format_of(:title).
    with(/\A.*[[:alpha:]]+.*\z/).
    message(N_('Please write a summary with some text in it')).
    unless(proc { |info_request| info_request.title.blank? })
```